### ISSUE TITLE
Moving or copying everything to news.git-scm.com

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#222]
At the Git Merge 2017 @tfnico and I talked to Peff about possibly moving or copying Git Rev News to something like news.git-scm.{com,org} (that is a subdomain of git-scm.{com,org}). Peff is ok with that and I think @tfnico wants to take a look at it.

**Comment by tfnico** [comment]
Yeah, so far I just tried moving the Jekyll stuff to the root of the repository, and that renders fine (you need to clone it and run it locally to see the result): https://github.com/tfnico/news.git-scm.com

It would be nice to have a bit fresher design if we "re-launch" news like this, so perhaps it would be better to take a modern Jekyll site, apply the git-scm.com styles, and then port the old contents in again.. Wouldn't let this block us though.

**Comment by peff** [comment]
Yeah, I think this is a fine idea. I agree with @tfnico that it would be nice to give it a more consistent graphical design with the rest of the git-scm.com site, but that it isn't strictly necessary.

I think this can be done entirely separately from the ongoing movement of the main site. The only blocker is that I am waiting to get access to the doman's DNS setup (I have a pending request in). I think I should be able to just "soft-launch" the site by pointing the domain at the GitHub page (and I suspect you'll need to tell GitHub Pages about the custom name, too; I've never set up a custom Pages domain, but there should be a guide).

You may eventually want to use a CDN, as I think you can't do custom-domain https without one. But that can happen later. The main site is probably going to need the same thing, so whatever setup we do there can probably be leveraged to cover both.

**Comment by tfnico** [comment]
@peff Is there any chance you can set up a redirect from news.git-scm.com to the current Git Rev News site? Then I could voice the URL already in the podcast intros.

**Comment by peff** [comment]
@tfnico Hrm. It turns out to be a little complicated. I have access to the domain registrar account for both git-scm.com and git-scm.org. The ".org" variant also has its DNS service hosted with the same account. But ".com" doesn't, and I don't have access to that account. In the long run, I think we will switch to host the DNS for both domains in the same place. And then it should be trivial to add a new entry.

I'm just a little hesitant to do that, though,  as I don't know what's _in_ the current ".com" zone file. It points to the Heroku site, and obviously I can reverse-engineer bits of it from DNS queries (and what Heroku tells me the DNS should look like). But it carries a risk to the main site. I was hoping to put that off until we decided what was happening to the main site (e.g., if it's getting moved to a static setup, or a CDN).

I did set up `news.git-scm.org` as a redirect to revnews, at least as a test. I'm not sure if you want to start advertising that, though. Having the ".org" work but not the ".com" will probably just confuse people.

So I dunno. Maybe it is time to make the DNS hosting switch so we can move forward with things like this.

**Comment by chriscool** [comment]
Yeah, I agree it is safer to do the DNS hosting switch before trying to point news.git-scm.com to Git Rev News and before advertising news.git-scm.com or news.git-scm.org. Thanks!
